ecosystem/gstreamer_plugin/gst_mtl_st20p_rx.h

```diff
@@ -78,6 +77,8 @@ struct _Gst_Mtl_St20p_Rx {
   gchar pixel_format[MTL_PORT_MAX_LEN];
   guint framebuffer_num;
   guint fps_n, fps_d;
+  GstVideoFormat format;
+  gboolean zero_copy;
```

Comment:
This section is for arguments 
if those are not passed down by the user its better to move them above

Reply:
Fixed in bedd0895579ff54984a2df2d2aed7191e446539a